src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
```

Comment:
The constant 'PRICE_PER_DAY' is well-named, making it clear that it represents the daily rental price.